circuits/cpp/src/aztec3/circuits/apps/contract.hpp

```diff
@@ -36,15 +36,14 @@ template <typename NCT> class Contract {
 
     std::map<std::string, Contract<NCT>> imported_contracts;
 
-    Contract<NCT>(std::string const& contract_name)
-        : contract_name(contract_name)
+    explicit Contract<NCT>(std::string const& contract_name) : contract_name(contract_name)
```

Comment:
This seems to be the only explicit constructor. I wonder why it chose to make this one explicit?

Reply:
Hm, not sure why it didn't change others too.... I can look into it.